### ISSUE TITLE
Highlighting Parent Indent Line

### ISSUE THREAD

**Issue by jbcoder** [harai/indent-guide-improved#41]
I'm curious as to why the parent line is highlighted. Maybe this isn't an issue as much as it may be just an inquiry. The indent guides that I'm used to highlight the line that you're on. So, you have code like this: 

```
<div>
    <ul>
        <li></li>
        <li></li>
        <li></li>
    <ul>
</div>
```

If your cursor is on the first line, there would be a guide going directly down below `div` to right above the closing `div`. However, I'm seeing the guide go before the element which highlights the parent--not the current element.

**Comment by harai** [comment]
I don't know which one is more natural for users, because the current function is the only highlight style I have ever used.  Are there any text editors which support such highlight style?

**Comment by ztrayner** [comment]
I think jbcoder is thinking something like this --> https://github.com/facelessuser/BracketHighlighter which highlights the current bracket or element you're on.

**Comment by jbcoder** [comment]
@harai You know, after you mentioned that I went and looked at all of the editors/IDEs that I've used and I couldn't even find a good example of what I was thinking. But I have used Netbeans and Brackets and I think I connected it in my head with a line between the elements instead of before. So, it may just be in my head. Maybe it would be cool to have an option for that though. I just can't get used to the line being on the parent element. haha
